src/Raven.Server/Web/System/AdminResourcesStudioTasksHandler.cs

```diff
@@ -0,0 +1,121 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="AdminResourcesStudioTasksHandler.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+using System;
+using System.Threading.Tasks;
+using Raven.Abstractions.Data;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Web.System
+{
+    public class AdminResourcesStudioTasksHandler : RequestHandler
+    {
+        [RavenAction("/admin/*/toggle-disable", "POST", "/admin/{resourceType:databases|fs|cs|ts}/toggle-disable?name={resourceName:string|multiple}&isDisabled={isDisabled:bool}")]
+        public Task PostToggleDisableDatabases()
```

Comment:
What do you think about http://issues.hibernatingrhinos.com/issue/RavenDB-4489 ?